.github/actions/bench/action.yml

```diff
@@ -92,3 +92,6 @@ runs:
         output-file-path: output.json
         github-token: ${{ inputs.gh_token }}
         auto-push: true
+        comment-on-alert: true
+        summary-always: true
```

Comment:
Hm wait, but when would this happen? We want it to happen in the PR, not after merge. And `store-results` will only be set after merge.

Reply:
I mislooked this part, so according to the documentation, `comment-on-alert` seemed to only work after merge indeed
https://github.com/benchmark-action/github-action-benchmark?tab=readme-ov-file#future-work

But there's still be warning and summary in CI workflow
https://github.com/potsrevennil/mlkem-c-aarch64/actions/runs/11890509403

However I encounter permission error in my own testing prs, I'll need to figure this out first, turning this back to draft for now